sonar-python-plugin/src/main/java/org/sonar/plugins/python/coverage/PythonCoverageSensor.java

```diff
@@ -95,19 +59,19 @@ public void execute(SensorContext context, Map<InputFile, Set<Integer>> linesOfC
     String baseDir = context.fileSystem().baseDir().getPath();
     Settings settings = context.settings();
 
+    LOG.info("Python unit test coverage");
```

Comment:
@vilchik-elena Why this change? In the default "INFO" mode, it seems rather confusing to display that line if no report is provided.